vignettes/pr_review_guidance.Rmd

```diff
@@ -88,6 +91,69 @@ knitr::include_graphics("./pr_review_actions.png")
+
+Following the installation of system dependencies, R, and package dependencies, each workflow checks  the integrity of a specific component of the admiral codebase.
+
+### [`check-templates.yml`](check-templates.yml)
+
+This workflow checks for issues within template scripts.  For example, in the admiral package there are several template scripts with admiral-based functions showing how to build certain ADaM datasets.   As we update the admiral functions, we want to make sure these template scripts execute appropriately.  Functions in the template scripts that are deprecated or used inappropriately will cause this workflow to fail. Click on the details button on a failing action provides information on the where the template is failing.  
+
+### [`code-coverage.yml`](code-coverage.yml)
+
+This workflow measures code coverage for unit tests and reports the code coverage as a percentage of the _total number of lines covered by unit tests_ vs. the _total number of lines in the codebase_.
+
+The [`covr`][covr] R package is used to calculate the coverage.
+
+Report summaries and badges for coverage are generated using a series of other GitHub Actions.
+
+### [`links.yml`](links.yml)
+
+This workflow checks whether URLs embedded in code and documentation are valid. Invalid URLs reults in workflow failures. This workflow uses [`lychee`][lychee] to detect broken links.   Occasionally this check will detect false positives of urls that look like urls.  To remedy, please add this false positive to the `.lycheeignore` file.   
+
+### [`lintr.yml`](lintr.yml)
+
+Static code analysis is performed by this workflow, which in turn uses the [`lintr`][lintr] R package.  Any [`.lintr`](../../.lintr) configurations in the repository will be by this workflow.
+
+### [`man-pages.yml`](man-pages.yml)
+
+This workflow checks if the manual pages in the `man/` directory of the package are up-to-date with ROxygen comments in the code.
+
+Workflow failures indicate that the manual pages are not up-to-date with ROxygen comments, and corrective actions are provided in the workflow log.
+
+### [`pkgdown.yml`](pkgdown.yml)
+
+Documentation for the R package is generated via this workflow. This workflow uses the [`pkgdown`](pkgdown) framework to generate documentation in HTML, and the HTML pages are deployed to the `gh-pages` branch.
+
+Moreover, an additional `Versions` dropdown is generated via the [][multi-version-docs] GitHub Action, so that an end user can view multiple versions of the documentation for the package.
```

Comment:
is `[][multi-version-docs]` correct - not seen a blank [] before so not sure how this renders

Reply:
So I just straight copied this from admiraltemplate without reading it too thoroughly.  I have now removed all the brackets/links in the workflows as they are not set up the same here.